NAM/dsp.h

```diff
@@ -63,44 +63,26 @@ class DSP
   //   DSP subclass implementation.
   virtual void finalize_(const int num_frames);
   double GetExpectedSampleRate() const { return mExpectedSampleRate; };
-  bool HasLoudness() { return mLoudness != TARGET_DSP_LOUDNESS; };
-  void SetNormalize(const bool normalize) { this->mNormalizeOutputLoudness = normalize; };
+  bool HasNormalization() { return mLoudness != TARGET_DSP_LOUDNESS; };
```

Comment:
Nit: this'll work, but I'm going to mess w/ this a little bit. (I prefer "loudness" since I'm going to remove the "target" from this and let downstream users of this library decide for themselves how loud they want to aim for.